packages/sources/src/api/createSource.js

```diff
@@ -78,6 +79,11 @@ export const doCreateSource = async (formData, sourceTypes) => {
 
         const [ endpointDataOut, applicationDataOut ] = await Promise.all(promises);
 
+        if (formData.cm_source) {
+            console.log('lets get this party started');
+            await patchSource({ id: sourceDataOut.id, ...formData.cm_source });
```

Comment:
I would rather keep `billing_source` and `credentials` separate for now to avoid creating inconsistencies.

We can make PRs, which will change these names afterwards. (It need to be changed here, in Sources-UI and in Sources-API for AWS and Openshift.)